src/services/administration/store/policy/type.ts

```diff
@@ -0,0 +1,10 @@
+import { PolicyDataModel } from '@/services/administration/iam/policy/lib/type';
+
+export interface PolicyState {
+	policyData: Partial<PolicyDataModel>|null;
+}
+
+export interface UpdatePolicyParams {
+	policyId: string;
+	updateParams: any;
+}
```

Comment:
alert에 이렇게 들어가있긴 하던데 

```suggestion
export interface UpdatePolicyParams extends Partial<PolicyDataModel> {
	policyId: string;
}
```

욜케 변경하면 안될까여 ㅎㅎ